libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -637,7 +681,8 @@ bool AC_PolyFence_loader::load_from_eeprom()
         Debug("Fence: Allocating %u bytes for points",
               (unsigned)(count * sizeof(Vector2f)));
         _loaded_offsets_from_origin = new Vector2f[count];
-        if (_loaded_offsets_from_origin == nullptr) {
+        _loaded_coordinates = new Vector2l[count];
```

Comment:
maybe rename "_loaded_coordinates" to "_loaded_points_lla"?

Reply:
Ack - updated